sql_patch.php

```diff
@@ -16,7 +16,20 @@
 $ignoreAuth = true; // no login required
 
 require_once('interface/globals.php');
+require_once('library/sql.inc');
 require_once('library/sql_upgrade_fx.php');
+require_once('version.php');
+
+$versionService = new \services\VersionService();
+
+$desiredVersion = new \entities\Version();
+$desiredVersion->setAcl($v_acl);
```

Comment:
Is there a way to not let this script set the acl since this is address somewhere else?